### PR TITLE
[PRTL-3040] add controlled study switch component

### PR DESCRIPTION
**Pull request by caravinci**
## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

## Description of Changes
Adds the Controlled Access switch (purple button), without any functionality.

### CODE REVIEW

File: src/packages/@ncigdc/components/ControlledAccessSwitch/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
```
Comment (samrichca) [inline src/packages/@ncigdc/components/ControlledAccessSwitch/index.js]:
```suggestion
```

File: src/packages/@ncigdc/containers/explore/ExplorePage.js [hunk 2]
```diff
@@ -193,6 +121,13 @@ const ExplorePageComponent = ({
   return (
     <SearchPage
       className="test-explore-page"
+      ControlledAccess={DISPLAY_DAVE_CA && [
```
Comment (samrichca) [inline src/packages/@ncigdc/containers/explore/ExplorePage.js]: I'm curious why this is an array?

Comment (caravinci) [reply src/packages/@ncigdc/containers/explore/ExplorePage.js]: in case we later on add more components there, removing the modal... bad idea?

Comment (samrichca) [reply src/packages/@ncigdc/containers/explore/ExplorePage.js]: oh ok, makes sense. I thought this was some fancy boolean logic

Comment (caravinci) [reply src/packages/@ncigdc/containers/explore/ExplorePage.js]: hahah I wish I was fancy.

**Pull request merged**